### ISSUE TITLE
Privacy concern for recommendation systems

### ISSUE THREAD

**Issue by embray** [hayj/Renewal#9]
We need to determine how much information about users are given to recommendation systems.

Currently users can provide personally identifying details such as their full name and e-mail address to the system.  All users also have an opaque user ID.

Should recommendation systems have access to a user's personally identifying information?  I would say "no", but I wanted to make sure that was the intent.

Technically, if users provide their location, this can also be used to identify them.  But that is a risk anyone takes when providing their location to a web service so they can enable it while understanding the risks (do most people understand this risk though?)

**Comment by hayj** [comment]
You right we'll list all necessary / wanted information on users for recommenders system. But for the first prototype the only info recommenders need is the **history of reads** (click-and-read events) as news ids. Later we'll provide also the history of what they didn't read (just scroll or click / quit). About the location I think it will be great for next version, but we'll see how to implement this as events or attached data to another event, don't know... and what about the RGPD / data agreements...
For static information like gender / age we'll see.

**Comment by embray** [comment]
Yes, I figured rec systems might be interested in information like gender/age and that they should have access to that (perhaps with user's agreement).

For location, users can either enter a static location, or enable dynamic location tracking.  In the latter case their location would be included with each event, I figured.
